prov/verbs/src/verbs_eq.c

```diff
@@ -861,6 +861,68 @@ vrb_eq_xrc_disconnect_event(struct vrb_eq *eq,
 	}
 }
 
+static inline int
```

Comment:
nit: I'd drop the inline and let the compiler decide as this is a long function (and the compiler will decide anyway since we're in a .c file...)